docs/src/main/java/org/geotools/tutorial/coverage/ImageTiler.java

```diff
@@ -0,0 +1,262 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2011, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This file is hereby placed into the Public Domain. This means anyone is
+ *    free to do whatever they wish with this file. Use it well and enjoy!
+ */
+//docs start prelim
+package org.geotools.tutorial;
```

Comment:
Incorrect package declaration fixed in https://github.com/geotools/geotools/commit/fe28f403c9008d39a70b8d5f1b12340b46b875c7 on master.